adapters/vidoomy/params_test.go

```diff
@@ -0,0 +1,56 @@
+package vidoomy
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// This file actually intends to test static/bidder-params/vidoomy.json
+//
+// These also validate the format of the external API: request.imp[i].ext.vidoomy
+
+// TestValidParams makes sure that the vidoomy schema accepts all imp.ext fields which we intend to support.
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderVidoomy, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected vidoomy params: %s", validParam)
+		}
+	}
+}
+
+// TestInvalidParams makes sure that the Vidoomy schema rejects all the imp.ext fields we don't support.
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderVidoomy, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"zoneId":"123"}`,
+}
+
+var invalidParams = []string{
+	``,
+	`null`,
+	`true`,
+	`5`,
+	`4.2`,
+	`[]`,
+	`{}`,
+	`{"notmid":"123"}`,
+	`{"zone_id":123}`,
```

Comment:
Could you please add a test with `zoneId` set to an empty string to test the min length constraint and also a test with `zoneId` set to a different type to test the string constraint. I see you have a test with `zone_id` set to a number, did you intend for this to test the `zoneId` property? As of now, it's testing an unspecified property which is already covered by the `notmid` case.

Reply:
fixed the typo in zone id, added empty `zoneId`